.azure-pipelines-templates/matrix.yml

```diff
@@ -60,6 +64,19 @@ jobs:
           artifact_name: "SGX_Perf"
           ctest_filter: "${{ parameters.test.perf.ctest_args }}"
 
+  # Performance (NoSGX)
+  # NOTE: CI doesn't support SGX and NoSGX perf tests at the same time.
```

Comment:
I don't think that's quite accurate, it's just that perf tests are run on an SGX build now. If the perf job built SGX + virtual + v8 app, there should be no difficulty in running all tests since they are decorated accordingly and won't collide.

What this would do is make the perf job take longer to build and run, but a possibility we've discussed if that's the case is to split that into two separate jobs, and a have a third dependent job do the plotting.

I think the main impact would be the sheer amount of metrics displayed in PRs, which would probably only be manageable with https://github.com/jumaffre/cimetrics/issues/75

Reply:
Do you want to deal with this in this PR? I'd prefer to sort this out in a separate PR given the size of it.

Reply:
No, I'm very happy for it to be separate, I do agree it's a large PR.

Reply:
Opened https://github.com/microsoft/CCF/issues/3302 for this.